src/plugins/ecs/ecs_compares.py

```diff
@@ -0,0 +1,303 @@
+import boto3
+import logging
+import pprint
+
+# get ecs data from boto call
+def ecs_check_versions(profile_name, region_name, cluster_name, slack_channel, env_code_name, the_config, role_arn):
+
+	service_versions = []
+
+	awsKeyId = None
+	awsSecretKey = None
+	awsSessionToken = None
+
+	cluster_list = []
+
+	#get list of clusters
+	if type(cluster_name) == list:
+		for c_items in cluster_name:
+			cluster_list.append(c_items)
+	else:
+		cluster_list.append(cluster_name)
+
+	print cluster_list
+
```

Comment:
Did you mean to leave this print statement in here? If so, do you want to preface it with something that tells the user what is being printed?